Sources_v3/WebSocketClient/Events/MessageEvents.swift

```diff
@@ -19,8 +19,8 @@ public struct MessageNewEvent<ExtraData: ExtraDataTypes>: EventWithUserPayload,
         cid = try response.value(at: \.cid)
         messageId = try response.value(at: \.message?.id)
         createdAt = try response.value(at: \.message?.createdAt)
-        watcherCount = try response.value(at: \.watcherCount)
-        unreadCount = try response.value(at: \.unreadCount)
+        watcherCount = try? response.value(at: \.watcherCount)
+        unreadCount = try? response.value(at: \.unreadCount)
```

Comment:
The commit message `Update `MessageNewEvent` to be decoded from response` doesn't help me to understand what you're doing in the commit

Reply:
This should also be covered by a test

Reply:
Initially it was `Update MessageNewEvent to be decoded from /sync response` but the endpoint was gone for some reason.

Will this name work?

Reply:
👍